omnibus/config/software/updater.rb

```diff
@@ -59,11 +59,16 @@
       mkdir "/usr/lib/systemd/system/"
       systemdPath = "/usr/lib/systemd/system/"
     end
+
     erb source: "datadog-updater.service.erb",
        dest: systemdPath + "datadog-updater.service",
        mode: 0644,
-       vars: { install_dir: install_dir, etc_dir: etc_dir}
+       vars: { install_dir: "/opt/datadog-packages/datadog-updater/stable", etc_dir: etc_dir}
```

Comment:
Could this be expressed as a relative path from `install_dir` instead?

Reply:
Not really. I renamed the variable to installer_dir, because it is not related to the location where we installe the installer we use to do the boostrap.

A similar path to `/opt/datadog-packages/datadog-agent`  is hardcoded for the systemd units of the agent a few lines under by the way